src/cache.ts

```diff
@@ -16,9 +16,8 @@ export const getDirectoryCache = async (): Promise<Cache> => {
 };
 
 export const DIRECTORY_CACHE_REQUEST = (hostname: string, prefix: string) => {
-	const normalizedPrefix = normalizePrefix(prefix);
-	const url = `https://${hostname}${normalizedPrefix}${PRIVATE_TOKEN_ISSUER_DIRECTORY}`;
-	return new Request(url);
+	const base = prefix || hostname;
+	return new Request(`https://${base}${PRIVATE_TOKEN_ISSUER_DIRECTORY}`);
```

Comment:
is the prefix always a valid hostname?

form the name, it seems it should be prepended to the hostname when building the base